dags/imagegen_devx/configs/jax_ss_config.py

```diff
@@ -79,3 +79,75 @@ def get_gke_maxtext_jax_ss_config(
       task_test_config=job_test_config,
       task_gcp_config=job_gcp_config,
   )
+
+
+def get_gke_maxdiffusion_jax_ss_config(
```

Comment:
what's the delta with `get_gke_maxtext_jax_ss_config`? can we re-use it with delta params? Thanks!